src/Foundatio.Redis/Queues/RedisQueue.cs

```diff
@@ -353,7 +353,7 @@ public RedisQueue(Builder<RedisQueueOptionsBuilder<T>, RedisQueueOptions<T>> con
                         workListName = _workListName,
                         queueName = _options.Name,
                         now,
-                        timeout = timeout.Ticks
+                        timeout = timeout.TotalSeconds
```

Comment:
It's expecting total seconds in the lua script: https://github.com/FoundatioFx/Foundatio.Redis/blob/master/src/Foundatio.Redis/Scripts/DequeueId.lua#L5-L6

Reply:
Good find @niemyjski